src/toolkits/activity_classification/ac_data_iterator.hpp

```diff
@@ -108,6 +109,15 @@ class data_iterator {
      */
     neural_net::shared_float_array weights;
 
+
+    /**
+     * An array with shape: (requested_batch_size, 
+     * 1, prediction_window * predictions_in_chunk, 1)
+     *
+     * Each row is the sequence of raw class labels (indices) corresponding every feature.
```

Comment:
I'm not sure I follow what "corresponding every feature" means. Maybe "sequence of raw class labels (indices) for each individual sample"? What do you think?

Reply:
That makes more sense!